code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -463,6 +463,11 @@
 /obj/item/borg/upgrade/modkit/indoors/modify_projectile(obj/projectile/kinetic/K)
 	K.pressure_decrease *= modifier
 
+// This is meant for Fauna Containment Specialists, who are primarily tasked to defeat fauna who make their way station-side.
+/obj/item/borg/upgrade/modkit/indoors/ert
```

Comment:
I think at this point it would be easier to just add a recharging gun that specifically does a lot of damage to fauna. This equally makes the ERT exceptional people killers as well.